modules/stochastic_tools/python/visualize_sobol.py

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/env python3
+#* This file is part of the MOOSE framework
+#* https://www.mooseframework.org
+#*
+#* All rights reserved, see COPYRIGHT for full restrictions
+#* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+#*
+#* Licensed under LGPL 2.1, please see LICENSE for details
+#* https://www.gnu.org/licenses/lgpl-2.1.html
+
+import os
+import argparse
+import pandas as pd
+import json
+import numpy as np
+import plotly
+from plotly import subplots
+import plotly.graph_objects as go
+import mooseutils
+
+def command_line_options():
+    """
+    Command-line options for statistics tool.
+    """
+    parser = argparse.ArgumentParser(description="Command-line utility for visualizing or tabulating sobol indices computed from the stochastic tools module.")
+
+    parser.add_argument('--text-table', dest='format', action='store_const', const=0)
+    parser.add_argument('--markdown-table', dest='format', action='store_const', const=1)
+    parser.add_argument('--latex-table', dest='format', action='store_const', const=2)
+    parser.add_argument('--bar-plot', dest='format', action='store_const', const=3)
+    parser.add_argument('--heatmap', dest='format', action='store_const', const=4)
+
+    parser.add_argument('filename', type=str, help="The Reporter data file to open, for sample 'foo_out.json' or 'foo_out_*.json'.")
+    parser.add_argument('-obj', '--object', default=None, type=str, help="StatisticsReporter object to consider, by default the first object found is used.")
+    parser.add_argument('-val', '--values', default=[], nargs='+', type=str, help="List of values to consider, by default all values are considered.")
+    parser.add_argument('-s', '--stat', default='TOTAL', type=str, help="Type of sobol index to output. Options: 'TOTAL' (default), 'FIRST_ORDER', 'SECOND_ORDER'.")
+    parser.add_argument('-ci', '--confidence-interval', dest='confidence_interval', default=None, nargs=2, type=str, help="Pair of confidence interval levels to use, by default the largest and smallest are used.")
+    parser.add_argument('-t', '--time', default=None, type=float, help="Time to consider, by default the last time is used.")
+    parser.add_argument('-n', '--names', default='{}', type=str, help="Map between the value name and display name in json format. Example: '{\"value1_name\" : \"Value One\", \"value2_name\" : \"Value Two\"}'")
+    parser.add_argument('-p', '--param-names', dest='param_names', nargs='+', default=[], type=str, help="List of names to display for the parameters.")
+    parser.add_argument('-nf', '--number-format', dest='number_format', default='.4g', type=str, help="The number format for tables.")
+    parser.add_argument('-o', '--output', default=None, type=str, help="File name to save figure or table")
+    parser.add_argument('--ignore-ci', dest='include_ci', action='store_false', help="Use this argument to ignore confidence intervals.")
```

Comment:
The "dest" doesn't match the argument. However, you don't need "dest" in any of these. `argparse` automatically converts "foo-bar" to "foo_bar"

Reply:
I didn't know that! That's cool. The `dest` for `--ignore-ci` would still be different. I can change this if it isn't standard practice?

Reply:
I would remove it, doesn't seem like it should be different.

Reply:
Nevermind, I'll just change it so it matches.